op-node/flags/flags.go

```diff
@@ -118,6 +118,13 @@ var (
 		}(),
```

Comment:
Unrelated, but because it's close, you can fix
```suggestion
Value: ptr.New(sync.CLSync),
```
using new package `op-service/ptr`.

Reply:
I changed it, but I don't understand what's the benefit of this package altogether?